components/encoder/incremental/incremental_encoder.go

```diff
@@ -158,8 +159,10 @@ func (e *Encoder) Reconfigure(
 	e.A = encA
 	e.B = encB
 	e.boardName = newConf.BoardName
+	e.board = board
```

Comment:
`e.board` is only ever used on this line and in `Start`, which is only called from within this function. Instead of storing it in the struct, just make it a local variable here that is passed to `Start` as a function argument. That way, we don't need to think through all the race conditions that might happen in the future if someone tries using `e.board` elsewhere.

Same suggestion about `e.interrupts` below.